File: Exercises/08-banking-app-1.java
class Account {
    String name;
    int number;
    float balance;
    Account(String name, int number) {
        this.name = name;
        this.number = number;
        this.balance = 0;
    }
    void deposit(float amount) {
        this.balance += amount;
    }
    void withdraw(float amount) {
        if(this.balance >= amount) {
            this.balance -= amount;
        }
    }
}
Account smith = new Account("Smith", 12345);
smith.deposit(100);
System.out.println(smith.balance);
smith.withdraw(75);
System.out.println(smith.balance);
smith.withdraw(50);
System.out.println(smith.balance);

File: Exercises/08-banking-app-1.py
class Account:
    def __init__(self, name, number):
        self.name = name
        self.number = number
        self.balance = 0
    def deposit(self, amount):
        self.balance += amount
    def withdraw(self, amount):
        if self.balance >= amount:
            self.balance -= amount
smith = Account("Smith", 12345)
smith.deposit(100)
print(smith.balance)
smith.withdraw(75)
print(smith.balance)
smith.withdraw(50)
print(smith.balance)